tests/AutoReview/ProjectFixerConfigurationTest.php

```diff
@@ -27,15 +28,19 @@
 {
     public function testCreate()
     {
-        /** @var \PhpCsFixer\Config $config */
+        // @var \PhpCsFixer\Config $config
```

Comment:
why that change?
are you aware that `//` is regular comment and any annotation inside it is actually not an annotation ?

Reply:
no idea why this happened, definitely not on purpose